ajax/libs/cropperjs/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "cropperjs",
+  "filename": "cropper.min.js",
+  "description": "JavaScript image cropper.",
+  "version": "0.7.0",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/fengyuanchen/cropperjs.git"
+  },
+  "homepage": "http://fengyuanchen.github.io/cropperjs",
```

Comment:
@dannyxx001 Please try to use https for github.io domain, most of them prefer https.